djl-spring-boot-console-sample/src/main/java/ai/djl/spring/examples/console/ConsoleApplication.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
The date just for initial creation. We usually don't change it when modify the file.
```suggestion
 * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```